src/cavendish_particle_tracks/_widget.py

```diff
@@ -9,6 +9,7 @@
 
 import glob
 
+import dask.array as da
```

Comment:
I wouldn't use a shorthand here. If you're using a `dask.array`, readers of the code will want to know that it's a `dask.array` and _not_ a `numpy.array`. _Especially_ for type hints.